src/main/scala-sbt-1.0/compat.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle
+
+import sbt._
+import sbt.Keys._
+
+import sbt.io.CopyOptions
+
+object compat {
+
+  lazy val packageDocSettings: Seq[Def.Setting[_]] = Seq(
+    packageDoc in Compile := {
+      val sourceFile = (baseDirectory in LocalRootProject).value / "README.md"
+      val targetFile = crossTarget.value / s"${name.value}-javadoc.jar"
+
+      IO.copy(Seq(sourceFile -> targetFile), CopyOptions().withOverwrite(true)).toSeq
```

Comment:
The 0.13 method is still available in sbt 1: https://github.com/sbt/io/blob/v1.0.0/io/src/main/scala/sbt/io/IO.scala#L587-L589

Reply:
`CopyOptions` is also available?

Reply:
No CopyOptions is sbt 1 only. What I meant is you could use

```scala
IO.copy(Seq(sourceFile -> targetFile), overwrite = true).toSeq
```

in both sbt 0.13 and sbt 1 and avoid the extra files/directories.

Reply:
Got it.

However, now I recall why I had to split them up into different pieces of code. `IO.copy` method signature are not compatible among sbt `0.13` and `1.x`.

* 0.13 (3 arguments):
https://github.com/sbt/sbt/blob/0.13/util/io/src/main/scala/sbt/IO.scala#L616
* 1.0 (4 arguments): 
https://github.com/sbt/io/blob/v1.0.0/io/src/main/scala/sbt/io/IO.scala#L587

Am I missing something?

Reply:
You're right. Sorry for the noise.

Reply:
No problem, thanks anyway :)